NuRadioReco/detector/RNO_G/analog_components.py

```diff
@@ -35,8 +45,8 @@ def load_amp_response(amp_type='rno_surface', path=os.path.dirname(os.path.realp
     amp_gain_f = interp1d(ff, amp_gain_discrete, bounds_error=False, fill_value=1)
     # all requests outside of measurement range are set to 0
 
-    def get_amp_gain(freqs):
-        amp_gain = amp_gain_f(freqs)
+    def get_amp_gain(temp, freqs):
```

Comment:
Can you give the temp variable a reasonable default value? That way old code still works if the temperature is not specified.

Reply:
Also having temp as the second argument would be better so we don't have to change older code